lib/pf/services/manager/keepalived.pm

```diff
@@ -59,7 +60,7 @@ sub generateConfig {
 
     $tags{'vrrp'} = '';
     $tags{'mysql_backend'} = '';
-    my @ints = uniq(@listen_ints,@dhcplistener_ints, map { $_->{'Tint'} } @portal_ints);
+    my @ints = uniq(@listen_ints,@dhcplistener_ints, map { $_->{'Tint'} } @portal_ints, @radius_ints);
```

Comment:
wrap the map in parentheses so its clearer what you are mapping and to make sure it only applies to what you want:
`uniq(@listen_ints,@dhcplistener_ints, (map { $_->{'Tint'} } @portal_ints, @radius_ints));`